<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QucsMgr</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="quitBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>601</width>
      <height>80</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="title">
     <string>project Location :</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="localLocationRadio">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>local</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="teamRadio">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>261</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>asicRF   $QUCS_REPOSITORY variable</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="myPath">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>22</y>
       <width>501</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>$HOME/qucs_personal</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="goBtn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>490</y>
      <width>491</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>GO ...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="projectLocation">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>431</width>
      <height>27</height>
     </rect>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="projectList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>591</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>629</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
